api-metastore/src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -79,6 +80,7 @@
 import static org.zalando.nakadi.domain.Feature.FORCE_EVENT_TYPE_AUTHZ;
 
 @Component
+@DependsOn({"storageService"})
```

Comment:
The `DependsOn` probably should be tackled in another PR, as this one is already too big